tests/test_product_page.py

```diff
@@ -0,0 +1,93 @@
+from page_forms.product_page import ProductPage
+from page_forms.login_page import LoginPage
+from page_forms.basket_page import BasketPage
+from utils.constants import LOGIN_URL, PROMO_URLS, PRODUCT_URL, LOGIN_TO_PRODUCT_URL
+import time
+import pytest
+
+
+class TestUserAddToBasketFromProductPage:
+
+    @pytest.fixture(autouse=True)
+    def setup(self, browser):
+        register_page = LoginPage(browser, LOGIN_URL)
+        register_page.open()
+        email = str(time.time()) + "@fakemail.org"
+        password = "kdoemo38hedk84"
```

Comment:
Hardcoded credentials! If it's a part of  fake test data, please move to test_data.py
If application credentials  -> pass as environment variables or cmd line arguments. Anyway, keeping credentials in code - bad practise